state/dynamodb.yaml

```diff
@@ -309,6 +325,60 @@ Resources:
       - {'Fn::ImportValue': !Sub '${ParentAlertStack}-TopicARN'}
       OKActions:
       - {'Fn::ImportValue': !Sub '${ParentAlertStack}-TopicARN'}
+  BackupVault:
+    Type: 'AWS::Backup::BackupVault'
+    Properties:
+      # AccessPolicy: # FIXME later
+      BackupVaultName: !Ref 'AWS::StackName'
+      # EncryptionKeyArn: # FIXME later
```

Comment:
@michaelwittig What is missing here?

Reply:
I use the default key now to avoid the issue of loosing backups if the kms key is deleted.

Reply:
👍